SB2018/2018-03-23.md

```diff
@@ -0,0 +1,15 @@
+---
+to: All Teams
+subject: Media Consent Forms
+---
+
+Hi all,
+
+In order for anyone to attend the competition, a Media Consent Form must be filled out. If students are over 18, they can fill out the forms themselves, however if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.
```

Comment:
> if students are under 18, it is a legal requirement that a parent/guardian signs the form for them.

This implies that it is a legal requirement that the form be signed, which isn't what you mean. What you mean is that the form may only be signed by the parent/guardian, rather than the young person.

I'm also not entirely sure how anyone would ever enforce that; certainly it's not a general case that signatures from minors don't count (see: young person' bank accounts).

I'm not sure how best to phrase this, though maybe just dropping the "it is a legal requirement that" part would work; with a little rewording that would give:
> if students are under 18, a parent/guardian must sign the form on their behalf.

Reply:
fixed in 20aae3da391634046833ebfa6d57a9ac2a41ec75